muxers/mplex/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 0.43.1
+
+- Rename types to match naming convention in [discussion 2174](https://github.com/libp2p/rust-libp2p/discussions/2174).
```

Comment:
There is a single type renamed in this PR, so something like this would be more immediately helpful for readers:
> Rename `MplexConfig` to `Config` for consistency with other crates